config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-
+  root 'home#index'
+  resources :posts, only: [:new, :create, :show] 
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
 
```

Comment:
comments in the routes file on the other hand can go, you'll be making edits to this file often and the presence of comments will largely serve to get in your way, and the documentation for routing in Rails is [quite nice](http://guides.rubyonrails.org/routing.html)